src/main/webapp/app/entities/submission.model.ts

```diff
@@ -30,6 +30,8 @@ export abstract class Submission implements BaseEntity {
     public durationInMinutes?: number;
 
     public results?: Result[];
```

Comment:
also document this one as the attribute that is initialized if the server returns some values

Reply:
I added documentation to it